src/pages/components/anchor.mdx

```diff
@@ -63,6 +63,17 @@ import AnchorExternalCode from '!!raw-loader!../../examples/anchor/AnchorExterna
   <AnchorExternal />
 </CodeExample>
 
+#### Security
+
+When linking to third-party content you may be [exposing your site to performance and security issues](https://web.dev/external-anchors-use-rel-noopener/).
+Garden provides two attributes by default for external anchors:
+
+- `target="_blank"`
+- `rel="noopener noreferrer"`
+
+These attributes ensure the third-party content is ran in a unique process
+and unable to access the current pages `window` object.
```

Comment:
Grammar:

- "...ensure the third-party content is ran in a unique process" vs "...ensure ~the~ third-party content runs in a unique process"
- "...current pages `window`" vs "...current page's `window`"

Reply:
Also:

> These attributes ensure third-party content runs in a unique process and _can't_ access the current page's window object.

Make sure those `'` get keyed in as proper apostrophes please!